TOPdeskPS/functions/Supporting Files/Set-TdBranch.ps1

```diff
@@ -0,0 +1,150 @@
+﻿function Set-TdBranch {
+    <#
+.SYNOPSIS
+    update a branch by id
+.DESCRIPTION
+    update branch
+    .PARAMETER BranchId
+    ID of the branch. See Get-TdBranch
+    .PARAMETER Name
+    Name of the branch
+    .PARAMETER Specification
+    Branch specification
+    .PARAMETER ClientReferenceNumber
+    Client Reference Number
+    .PARAMETER Phone
+    Phone Number
+    .PARAMETER Fax
+    Fax Number
+    .PARAMETER Email
+    email address
+    .PARAMETER Website
+    Website URL
+    .PARAMETER BranchType
+    Define the type of branch. Optional values: 'independentBranch', 'headBranch', 'hasAHeadBranch'
+    .PARAMETER HeadBranchId
+    ID of head branch
+    .PARAMETER Address
+    Hashtable containing address values
+    .PARAMETER PostalAddress
+    Hashtable containing postal address values
+    .PARAMETER OptionalFields1
+    optional see docs
+    .PARAMETER OptionalFields2
+    optional see docs
+.PARAMETER Confirm
+    If this switch is enabled, you will be prompted for confirmation before executing any operations that change state.
+.PARAMETER WhatIf
+    If this switch is enabled, no actions are performed but informational messages will be displayed that explain what would happen if the command were to run.
+.EXAMPLE
+    PS C:\> Set-TdBranch -BranchId (Get-TdBranch -Name 'Test Branch').id -Phone '555-555-5555'
+    Updates the Test Branch phone number.
+#>
+    #TODO Update help
```

Comment:
agree. let's add some different examples. Perhaps we can show a quick way to generate them from AD.

Reply:
actually, lets not